sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Solved, now passes in Travis. The node-gyp compilation started working, when Travis instructed to use g++4.8 as the C compiler suite (CXX).